test/Microsoft.SqlTools.ServiceLayer.PerfTests/Tests/IntellisenseTests.cs

```diff
@@ -202,7 +202,7 @@ public class IntellisenseTests
         private static async Task ValidateCompletionResponse(TestHelper testHelper, string ownerUri, string query, [CallerMemberName] string testName="")
         {
             TestTimer timer = new TestTimer();
-            await Common.ExecuteWithTimeout(timer, 60000, async () =>
+            await Common.ExecuteWithTimeout(timer, 500000, async () =>
```

Comment:
Keep in mind this will keep the test running, even if it fails, for ~8min. I'm guessing that's the expected behavior, but it seems a little excessive for something we'd expect to complete in under a minute